.github/workflows/deliver.yml

```diff
@@ -155,3 +155,58 @@ jobs:
           files: /tmp/farmOS-${{ env.FARMOS_VERSION }}.tar.gz
           draft: false
           prerelease: false
+  publish:
+    name: Publish to Docker Hub
+    # We only publish to Docker Hub if this is a tag or 2.x branch push event
+    # to the official repository.
+    if: |
+      github.repository == 'farmOS/farmOS' &&
```

Comment:
Could also add a check to make sure the docker username/password are available before continuing